tests/integration/targets/docker_image/tasks/tests/docker_image.yml

```diff
@@ -136,6 +136,10 @@
     - repository_1 is changed
     - repository_2 is not changed
 
+- assert:
+    that:
+    - repository_1.stdout
```

Comment:
Please note that the tests have been deactivated since development of the docker modules moved to https://github.com/ansible-collections/community.docker/ You have to run the tests manually to find out whether this actually works.

Also, a slightly more helpful test would be good. Is there something that always appears in the output? You should check whether it's contained in `stdout` then.

Reply:
Oh should this change then go here or in that collection?